OPHD/UI/Core/UIContainer.h

```diff
@@ -11,7 +11,7 @@
  * 
  * Generally not intended to be used by itself.
  */
-class UIContainer: public Control
+class UIContainer: public TextControl
```

Comment:
This one caught me by surprise since a generic `UIContainer` doesn't inherently support text operations. Though the `Window` class does derives from `UIContainer`, and the `Window` class has text for the window title.

My first thought was to add the text support only to `Window`, but there doesn't seem to be a way to do that without introducing multiple inheritance. That's something I'd like to avoid for the time being.

I suppose this is a reasonable way to organize things for now. No better solution comes to mind.